sys/include/net/dhcpv6/client.h

```diff
@@ -59,6 +59,20 @@ extern "C" {
 #define CONFIG_DHCPV6_CLIENT_PFX_LEASE_MAX (1U)
 #endif
 
+/**
+ * @brief   Maximum number of address leases to be stored
+ *
+ * @note    Used for calculation of @ref CONFIG_GNRC_NETIF_IPV6_ADDRS_NUMOF.
+ *          Set to 0 if `dhcpv6_client_ia_na` is not included.
+ */
+#ifndef CONFIG_DHCPV6_CLIENT_ADDR_LEASE_MAX
+#if defined(MODULE_DHCPV6_CLIENT_IA_NA) || defined(DOXYGEN)
+#define CONFIG_DHCPV6_CLIENT_ADDR_LEASE_MAX (1U)
+#else
+#define CONFIG_DHCPV6_CLIENT_ADDR_LEASE_MAX (0U)
+#endif
+#endif
```

Comment:
Arghs, now the reverse happened, but in `gnrc_netif` (which was not changed in this PR)... so I'd say revert the last change and...
```suggestion
#ifndef CONFIG_DHCPV6_CLIENT_ADDR_LEASE_MAX
#if defined(MODULE_DHCPV6_CLIENT_IA_NA) || defined(DOXYGEN)
#define CONFIG_DHCPV6_CLIENT_ADDR_LEASE_MAX (1)
#else
#define CONFIG_DHCPV6_CLIENT_ADDR_LEASE_MAX (0)
#endif
#endif
```

Reply:
This PR is very stubborn ;) I replaced the code and took the freedom of squashing immediately.

Reply:
Stubborn indeed :-/